src/test/ui/consts/miri_unleashed/abi-mismatch.stderr

```diff
@@ -4,7 +4,7 @@ error[E0080]: could not evaluate static initializer
 LL |     my_fn();
    |     ^^^^^^^
    |     |
-   |     calling a function with ABI C using caller ABI Rust
+   |     calling a function with calling convention C using calling convention Rust
```

Comment:
Oh, this is nice, this alleviates one of my concerns that this could allow `extern "C" fn` and (`extern "Rust" `)`fn`s to be considered equivalent.

The funniest part is I forgot it happened, but `git blame`s https://github.com/rust-lang/rust/commit/79d908b301c4ad8b0778331af4b0c6ad46970ef1.

Reply:
> (EDIT: oh I see, I think the reason for that commit was this part - nice that it helps miri years later)

That looks strange though, considering that `FnAbi` contains `can_unwind`. Or maybe that's a change that happened since then, since I cannot find that code in the current tree any more anyway.